App/Controllers/SummonersController.php

```diff
@@ -42,17 +43,44 @@ public function store(ServerRequest $request)
         $divisionId = intval($division->first()->getAttributes()['id']);
 
         // @todo replace lane with actual aggregated data
-        $lane = $leagueOfLegendsService->matchlist($summonerData['summoner_id']);
+        $matchList = $leagueOfLegendsService->matchlist($summonerData['summoner_id']);
```

Comment:
Let's discuss creating a new class or two here. We know we want the match list, but below we are doing a bunch of mutation (unsetting, changing lane casing). Right now we are having LeagueWrapper classes coming out of _our_ wrapper. Instead we could mutate the matchlist within the service and output our own class that guarantees the data does not need further mutation.